Azure/azure-cli
# {packaging} Update jsondiff to 1.3.0

**Description**
The `jsondiff` package was released in April 2021 and it works well with `azure-cli`. The current `azure-cli` package in Fedora 35/36 uses `jsondiff` 1.3.0 without any issues or testing failures.

**Testing Guide**
Existing tests should continue working without modifications.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
